### PR TITLE
Internal documentation for tracing-related API.

### PR DESCRIPTION
**Pull request by sumerman**
## Description

- This PR partially addresses #159 by adding SQL comments to functions and procedures that comprise Tracing API.
- Additionally, I've noticed that `(tag_v,tag_v)` eq and neq functions lacked the support function annotation, so I've added it.

## Merge requirements

Please take into account the following non-code changes that you may need to make with your PR:

- [ ] ~CHANGELOG entry for user-facing changes~
- [x] Updated the relevant documentation

### CODE REVIEW

File: migration/idempotent/008-tracing-functions.sql [hunk 1]
```diff
@@ -28,6 +32,8 @@ AS $sql$
 $sql$
 LANGUAGE SQL IMMUTABLE PARALLEL SAFE;
 GRANT EXECUTE ON FUNCTION ps_trace.event_tag_type() TO prom_reader;
+COMMENT ON FUNCTION ps_trace.event_tag_type 
+IS 'This function returns tag_type with the event tag bit.';
```
Comment (JamesGuthrie) [inline migration/idempotent/008-tracing-functions.sql:36]:
```suggestion
IS 'This function returns tag_type with the event tag bit set.';
```

File: migration/idempotent/008-tracing-functions.sql [hunk 2]
```diff
@@ -37,6 +43,8 @@ AS $sql$
 $sql$
 LANGUAGE SQL IMMUTABLE PARALLEL SAFE;
 GRANT EXECUTE ON FUNCTION ps_trace.link_tag_type() TO prom_reader;
+COMMENT ON FUNCTION ps_trace.link_tag_type
+IS 'This function returns tag_type with the link tag bit.';
```
Comment (JamesGuthrie) [inline migration/idempotent/008-tracing-functions.sql:47]:
```suggestion
IS 'This function returns tag_type with the link tag bit set.';
```

File: migration/idempotent/008-tracing-functions.sql [hunk 3]
```diff
@@ -19,6 +21,8 @@ AS $sql$
 $sql$
 LANGUAGE SQL IMMUTABLE PARALLEL SAFE;
 GRANT EXECUTE ON FUNCTION ps_trace.resource_tag_type() TO prom_reader;
+COMMENT ON FUNCTION ps_trace.resource_tag_type
+IS 'This function returns tag_type with the resource tag bit.';
```
Comment (JamesGuthrie) [inline migration/idempotent/008-tracing-functions.sql:25]:
```suggestion
IS 'This function returns tag_type with the resource tag bit set.';
```

File: migration/idempotent/008-tracing-functions.sql [hunk 4]
```diff
@@ -10,6 +10,8 @@ AS $sql$
 $sql$
 LANGUAGE SQL IMMUTABLE PARALLEL SAFE;
 GRANT EXECUTE ON FUNCTION ps_trace.span_tag_type() TO prom_reader;
+COMMENT ON FUNCTION ps_trace.span_tag_type
+IS 'This function returns tag_type with the span tag bit.';
```
Comment (JamesGuthrie) [inline migration/idempotent/008-tracing-functions.sql:14]:
```suggestion
IS 'This function returns tag_type with the span tag bit set.';
```

File: migration/idempotent/008-tracing-functions.sql [hunk 5]
```diff
@@ -421,6 +466,10 @@ AS $func$
 $func$
 LANGUAGE SQL STABLE PARALLEL SAFE STRICT;
 GRANT EXECUTE ON FUNCTION ps_trace.get_tag_map(jsonb) TO prom_reader;
+COMMENT ON FUNCTION ps_trace.get_tag_map
+IS 'For a given jsonb object cosisting of key-value pairs, representing tags and their values,
```
Comment (JamesGuthrie) [inline migration/idempotent/008-tracing-functions.sql:470]:
```suggestion
IS 'For a given jsonb object consisting of key-value pairs, representing tags and their values,
```

File: migration/idempotent/008-tracing-functions.sql [hunk 6]
```diff
@@ -421,6 +466,10 @@ AS $func$
...
+this funciton returns a jsonb object of corresponding ids -- the primary keys in
```
Comment (JamesGuthrie) [inline migration/idempotent/008-tracing-functions.sql:471]:
```suggestion
this function returns a jsonb object of corresponding ids -- the primary keys in
```

**Review by JamesGuthrie — CHANGES_REQUESTED** [review]

**Comment by sumerman** [comment]
Thank you @JamesGuthrie for reading through this!

**Review by JamesGuthrie — APPROVED** [review]

**Pull request merged**